### PR TITLE
Publish SCAP scan results; Improve Windows logging

### PR DESCRIPTION
**Pull request by YakDriver**

### CODE REVIEW

**Comment by YakDriver** [comment]
Two new input vars: `wam_version` and `s3_scan_bucket`. If `wam_version` exists, Terrafirm will publish the scan results to the `s3_scan_bucket`. The instance profile must have access to the scan bucket.

File: templates/win_userdata.ps1 [hunk 1]
```diff
@@ -254,17 +269,15 @@ function Install-PythonGit {
...
   & "$BootstrapFile" `
-      -PythonUrl "$PythonUrl" `
-      -GitUrl "$GitUrl" `
-      -Verbose -ErrorAction Stop
-  Test-DisplayResult "Install Python/Git" $?
+    -PythonUrl "$PythonUrl" `
+    -GitUrl "$GitUrl" `
+    -Verbose -ErrorAction Stop
+  $Script = "$BootstrapFile -PythonUrl $PythonUrl -GitUrl $GitUrl -Verbose -ErrorAction Stop"    
+  Test-DisplayResult "Install Python/Git [$Script]" $?
```
Comment (lorengordon) [inline templates/win_userdata.ps1:280]: won't `$?` here capture the result of the `$Script` assignment on the prior line, not the return code from the bootstrap script?

Comment (YakDriver) [reply templates/win_userdata.ps1]: Yes. I fixed it. I've tried several things to make that line work with one string that gets executed and logged but none have worked. That was a missed artifact of those attempts.

**Comment by lorengordon** [comment]
go codebuild go

**Comment by lorengordon** [comment]
:shipit:!

**Pull request merged**